ServiceTier/Services/TaskService.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Timer.DAL.Extensions;
+using Timer.DAL.Timer.DAL.Entities;
+using Timer.DAL.Timer.DAL.UnitOfWork;
+
+using Common.Exceptions;
+using Common.ModelsDTO;
+
+namespace ServiceTier.Services
+{
+    public class TaskService : BaseService, ITaskService
+    {
+        public TaskService(IUnitOfWork unitOfWork) : base(unitOfWork)
+        {
+
+        }
+
+        public void CreateTask(TaskDTO taskDTO)
+        {
+            var task = taskDTO.TaskDTOToTask();
+            unitOfWork.Tasks.Create(task);
+            unitOfWork.Save();
+        }
+
+        public void DeleteTaskById(int taskId)
+        {
+            var toDelete = unitOfWork.Tasks.GetByID(taskId);
+            if (toDelete != null)
+            {
+                unitOfWork.Tasks.Delete(toDelete);
+                unitOfWork.Save();
+            }
+            else
+            {
+                throw new TaskNotFoundException();
+            }
+        }
+
+        public IQueryable<TaskDTO> GetAllTasks()
```

Comment:
service should never return IQueriable. Change to IEnumerable and we'll talk more about it on meetup.